engine/src/main/junit-shared/org/operaton/bpm/engine/impl/test/TestHelper.java

```diff
@@ -564,4 +565,14 @@ public static void executeJobIgnoringException(ManagementService managementServi
       // do nothing
     }
   }
+
+  public static void executeJobExpectingException(ManagementService managementService, String jobId) {
+    executeJobExpectingException(managementService, jobId, Exception.class);
```

Comment:
Can we assume that the exception is `ProcessEngineException`. `AuthorizationException` is also a subclass. I think any other RuntimeException should be unexpected and make the caller fail.